cypress/integration/sGMR/voyage-report-form-validation.spec.js

```diff
@@ -0,0 +1,160 @@
+const faker = require('faker');
+const { getFutureDate, getPastDate } = require('../../support/utils');
+
+describe('Validate report form', () => {
+  let departureDateTime;
+  let departurePort;
+  let arrivalDateTime;
+  let arrivalPort;
+  let vessel;
+  let people;
+
+  before(() => {
+    cy.task('readFileMaybe', 'cypress/fixtures/users.json').then((data) => {
+      const user = JSON.parse(data);
+      cy.checkUserExists(user.email).then((userExist) => {
+        if (userExist === false) {
+          cy.registerUser();
+        }
+      });
+    });
```

Comment:
I think we should use the new syntax :)

```suggestion
    cy.registerUser();
```

Reply:
Yes making changes ! thanks